test/metabase/query_processor/middleware/add_source_metadata_test.clj

```diff
@@ -411,3 +408,149 @@
                 :query
                 :source-metadata
                 (map :field_ref))))))
+
+#_(deftest ^:synchronized deleted-columns-test
```

Comment:
Let's either delete this if it's dead code, or add a TODO about why it's disabled and when it can be brought back.

Reply:
oh yeah oops I copied that test from a different namespace and I was going to adapt it to this namespace as well but I forgot about that. I'll just delete it.